src/splitInteger.test.js

```diff
@@ -4,18 +4,22 @@ const splitInteger = require('./splitInteger');
 
 test(`should split a number into equal parts
```

Comment:
It's a good practice to add a `describe` block to wrap and group your tests. This will make your test suite more organized and readable. 😊